Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -1,24 +1,28 @@
 #include "attitudeStateClasses.hpp"
+#include "Controls.hpp"
 #include "PWM.hpp"
 #include "safetyConfig.hpp"
 #include "RSSI.hpp"
-<<<<<<< HEAD
 #include "PID.hpp"
-=======
 #include "PPM.hpp"
->>>>>>> 2ac372122bb825aa8f51705d39eab6195a5f3817
-
+// #include "CommFromPMToAM.hpp"
+#include "../../boardfiles/Drivers/STM32F0xx_HAL_Driver/Inc/stm32f0xx_hal.h"
 
 /***********************************************************************************************************************
  * Definitions
  **********************************************************************************************************************/
 
 float OutputMixingMode::_channelOut[4];
 SFOutput_t sensorFusionMode::_SFOutput;
-PID_Output_t PIDloopMode::_PidOutput;
+PID_Output_t *PIDloopMode::_PidOutput;
+Instructions_t *_ControlsInstructions;
 PWMChannel pwm; 
+PPMChannel ppm;
```

Comment:
Is the constructor PPMChannel(uint8_t channels, uint32_t timeout) ever called?

Reply:
I don't think so, however, in the original code that wasn't done either so not sure if its needed or not